fix: Detem registro de usuários com permissões acima do requisitante

PaulaBelo opened the pull request:
Impede o registro de usuários com permissão superior ao que está efetuando a requisição, retornando uma mensagem de erro ao solicitar tal rota.

PaulaBelo committed:
fix: Detem registro de usuários com permissões acima do requisitante

Impede o registro de usuários com permissão superior ao que está efetuando a requisição, retornando uma mensagem de erro ao solicitar tal rota.

LucasTonetto commented on src/ts/routes/register.ts:
```diff
@@ -63,6 +63,19 @@ const register = (app: { [key: string]: any }): void => {
+				(req.permission === 'adopsteammanager' && req.body.permission === 'admin') ||
+				(req.permission === 'adopsteammanager' && req.body.permission === 'owner')
+			) {
+				const message = 'Usuário sem permissão para tal rota!';
```
Usuário sem permissão para criar um novo usuário de permissão {{nivel da permissão que está tentando ser criada}}

PaulaBelo committed:
fix: Correção da frase de erro de registro e alteração nas declarações de AdOpsManager

Correção/Mudança na frase de erro de registro com o retorno da permissão que o requisitante não pode criar. Alteração em todas as declarações de AdOpsManager que estavam dando divergência no código.
